Makefile

```diff
@@ -111,15 +111,13 @@ help: ## Display this help.
 # Directories.
 
 ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
-BIN_DIR := bin
-TOOLS_DIR := hack/tools
-TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/$(BIN_DIR))
+BIN_DIR := $(abspath bin)
```

Comment:
cc @msherif1234 we already had all our executable tools in `./bin` (e.g. operator-sdk ...) , so this is moving `conversion-gen` to the same place to avoid too much fragmentation

Reply:
Sure make sense